installer/debian/algorand/postinst

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Hmm, do we have to use bash here? That adds a dependency.

Reply:
I changed it to use bash for the brace expansion that you commented on below.  If this is a concern, and it would be for alpine, I can set it back to use the bourne shell.

Reply:
Because we don't know all the debian variants, if we can keep to native bourne shell, I'd vastly prefer it.

Reply:
Will fix.